### PR TITLE
Add region to CloudTrail DigestError message

### PR DESCRIPTION
**Pull request by JordonPhillips**
I additionally cleaned up some minor formatting / unused parameter issues. Seperated commits for improved readability.

cc @jamesls @kyleknap

### CODE REVIEW

File: awscli/customizations/cloudtrail/validation.py [hunk 1]
```diff
@@ -486,8 +488,10 @@ def _load_and_validate_digest(self, public_keys, bucket, key):
         fingerprint = digest_data['digestPublicKeyFingerprint']
         if fingerprint not in public_keys:
             raise DigestError(
-                ('Digest file\ts3://%s/%s\tINVALID: public key not found for '
-                 'fingerprint %s') % (bucket, key, fingerprint))
+                ('Digest file\ts3://%s/%s\tINVALID: public key not found in %s '
```
Comment (kyleknap) [inline awscli/customizations/cloudtrail/validation.py]: Maybe make it ``"in region %s"`` so they know that you are referring to a region?

**Comment by kyleknap** [comment]
Looks good but had one comment about the messaging. Otherwise :ship:

**Comment by JordonPhillips** [comment]
👍

**Comment by jamesls** [comment]
:shipit: Assuming travis build is good.

**Pull request merged**